tests/conftest.py

```diff
@@ -153,14 +155,16 @@ async def db():
 
 
 @pytest_asyncio.fixture
-async def insert_fake_resource():
-    async def _insert_fake_resource(database) -> None:
-        await database.execute(
-            f"""
-            INSERT INTO catalog (dataset_id, resource_id, url, priority, deleted)
-            VALUES ('{DATASET_ID}', '{RESOURCE_ID}', 'http://dev.local/', True, False)
-            ON CONFLICT (resource_id) DO NOTHING;
-            """
+async def insert_fake_resource(status: Union[str, None] = None):
```

Comment:
I'm not sure the parameters are useful/used in this outer fixture function? I believe they could be simply removed, right?

Reply:
You mean the `status` parameter? It's not used yet, but I added it because I might use it later.

Reply:
How would you use this `status` parameter on the fixture itself?

When using a fixture in a test, you use it like that, right?
```python
def test_foo(insert_fake_resource):
    insert_fake_resource("TO_CHECK")
```

But there's no parameter passing here, you can't do
```python
def test_foo(insert_fake_resource("TO_CHECK")):
    insert_fake_resource()
```

So i understand the parameter in the inner function (the helper, the one returned by the fixture), but not the parameter in the fixture function itself.

Am I missing something?

Reply:
No, you're right, just fixed! Thanks